clover_admin.rb

```diff
@@ -182,22 +182,104 @@ def self.object_action(...)
     form_options(wrapper: :div)
 
     link = lambda do |obj|
+      return "" unless obj
       "<a href=\"/model/#{obj.class}/#{obj.ubid}\">#{Erubi.h(obj.name)}</a>"
     end
 
     show_html do |obj, column|
       case column
       when :name
         link.call(obj)
-      when :project, :location
+      when :project, :location, :vm_host
         link.call(obj.send(column))
       end
     end
 
+    model Account do
+      order Sequel.desc(Sequel[:accounts][:created_at])
+      eager [:identities]
+      columns [:name, :email, :status_id, :provider_names, :created_at, :suspended_at]
+      column_options email: {type: "text"},
+        status_id: {type: "select", options: {Unverified: 1, Verified: 2, Closed: 3}, add_blank: true},
+        provider_names: {label: "Providers", type: "select", options: ["google", "github"], add_blank: true},
+        created_at: {type: "text"},
+        suspended_at: {label: "Suspended", type: "boolean", value: nil}
+
+      column_search_filter do |ds, column, value|
+        if column == :provider_names
+          ds.join(:account_identities, account_id: :id).where(provider: value)
+        elsif column == :created_at
+          ds.where(Sequel.cast(Sequel[:accounts][:created_at], :text).ilike("%#{ds.escape_like(value)}%"))
+        elsif column == :suspended_at
+          ds.send((value == "t") ? :exclude : :where, suspended_at: nil)
+        end
+      end
+    end
+
     model Firewall do
       eager [:project, :location]
       columns [:name, :project, :location, :description]
     end
+
+    model GithubInstallation do
+      order Sequel.desc(:created_at)
+      columns [:name, :installation_id, :type, :cache_enabled, :premium_runner_enabled?, :created_at, :allocator_preferences]
+
+      column_options type: {type: "select", options: ["Organization", "User"], add_blank: true},
+        premium_runner_enabled?: {label: "Premium enabled", type: "boolean", value: nil},
+        created_at: {type: "text"}
+
+      column_search_filter do |ds, column, value|
+        if column == :premium_runner_enabled?
+          ds.where((value == "t") ?
+            Sequel.lit("allocator_preferences->'family_filter' @> ?", Sequel.pg_jsonb(["premium"])) :
+            Sequel.lit("allocator_preferences->'family_filter' IS NULL OR NOT (allocator_preferences->'family_filter' @> ?)", Sequel.pg_jsonb(["premium"])))
+        elsif column == :allocator_preferences
+          ds.where(Sequel.cast(:allocator_preferences, :text).ilike("%#{ds.escape_like(value)}%"))
+        elsif column == :created_at
+          ds.where(Sequel.cast(:created_at, :text).ilike("%#{ds.escape_like(value)}%"))
+        end
```

Comment:
DRYer to use a `case` statement and the Sequel pg_json_ops extension support here:

```suggestion
        case column
        when :premium_runner_enabled?
          family_filter = Sequel.pg_jsonb(:allocator_preferences).get('family_filter')
          cond = family_filter.contains(["premium"])
          if value == "t"
            ds.where(cond)
          else
            ds.where(~cond | {family_filter => nil})
          end
        when :allocator_preferences
          ds.where(Sequel.cast(:allocator_preferences, :text).ilike("%#{ds.escape_like(value)}%"))
        when :created_at
          ds.where(Sequel.cast(:created_at, :text).ilike("%#{ds.escape_like(value)}%"))
        end
```